### ISSUE TITLE
Add Resources to Init Containers

### ISSUE THREAD

**Issue by norwoodj** [aws/aws-app-mesh-inject#59]
We deploy ResourceQuotas in each of our application namespaces, in order to force teams to put memory/CPU requests on all of their pods. As a result, our injected pods cannot start, because the `proxyinit` initContainer does not specify any resources requests or limits. We request that the init container has resources and limit specified.

**Comment by stefanprodan** [comment]
Currently we set only resources requests for the Envoy sidecar, this could be ported to the init container. As for limits, I don't see how we can set those for Envoy, it would cause OOM on workloads under load.

**Comment by nckturner** [comment]
@stefanprodan agreed, but allowing optional (default disabled) limits shouldn't be a problem but still allow that option for users who require it, right?
